Course/Course/Presentation/Outline/CourseOutlineView.swift

```diff
@@ -271,6 +273,33 @@ public struct CourseOutlineView: View {
             }
         }
     }
+    
+    @ViewBuilder
+    private var certificateView: some View {
+        // MARK: - Course Certificate
+        if let certificate = viewModel.courseStructure?.certificate {
+            if let url = certificate.url, url.count > 0 {
```

Comment:
May be we can also refactor it like this to avoid nested if let conditions:
```
if let certificate = viewModel.courseStructure?.certificate,
   let url = certificate.url,
   url.count > 0 {
```

Reply:
Sure, you are right. Fixed. Thanks.